diff --git a/tensorflow_serving/model_servers/BUILD b/tensorflow_serving/model_servers/BUILD
index f60f3d77..d0c709ff 100644
--- a/tensorflow_serving/model_servers/BUILD
+++ b/tensorflow_serving/model_servers/BUILD
@@ -296,6 +296,7 @@ load("//tensorflow_serving:tensorflow_version.bzl", "if_v2", "if_not_v2")
 SUPPORTED_TENSORFLOW_OPS = if_v2([]) + if_not_v2([
     "@org_tensorflow//tensorflow/contrib:contrib_kernels",
     "@org_tensorflow//tensorflow/contrib:contrib_ops_op_lib",
+    "//tensorflow_serving/neuron/runtime:all_ops",
 ])
 
 TENSORFLOW_DEPS = [
